### PR TITLE
move all autoload and internal resources to woof.pk3 archive

### PR DESCRIPTION
**Pull request by rfomin**
* Remove -dumplumps and -dumptables commands.

### CODE REVIEW

File: base/all-all/woofhud.lmp [hunk 1]
```diff
@@ -28,6 +26,5 @@ weapon bottomright
 keys bottomleft
 monsec bottomleft
 sttime bottomleft
-coord topright
+coord  topright
 fps topright
-cmd bottomright
```
Comment (ceski-1) [inline base/all-all/woofhud.lmp:33]: It looks like this needs to be the lump from the master branch.

**Comment by fabiangreffrath** [comment]
I think it would make sense if `lumpinfo[].wad_file` would point to the "container" that a lump was loaded in, i.e. to the lump itself if it was loaded on the command line, to the ZIP file if it was packed in a ZIP or to the WAD file if it was loaded from a WAD (what if this in turn was loaded from a ZIP?).

**Comment by rfomin** [comment]
> I think it would make sense if `lumpinfo[].wad_file` would point to the "container" that a lump was loaded in, i.e. to the lump itself if it was loaded on the command line, to the ZIP file if it was packed in a ZIP or to the WAD file if it was loaded from a WAD (what if this in turn was loaded from a ZIP?).

Currently, `lumpinfo[].wad_file` only points to the name of the wad in the `wadfiles` array (whether it's from zip) or `NULL`. Perhaps we should implement a more complex system, but I think that is beyond the scope of this PR.

**Comment by fabiangreffrath** [comment]
Okay, so `W_WadNameForLump()` still does the right thing.

**Comment by fabiangreffrath** [comment]
Is `wadfiles[0]` still the IWAD or is this now `woof.pk3`? I am asking for `W_IsIWADLump()`.

**Comment by rfomin** [comment]
> Is `wadfiles[0]` still the IWAD or is this now `woof.pk3`?

`wadfile[0]` is IWAD.

**Comment by rfomin** [comment]
If there's no objections I'd like to merge this.

**Review by fabiangreffrath — APPROVED** [review]
Thank you!

**Pull request merged**